### ISSUE TITLE
Mesangial proliferative lupus nephritis

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6833]
**Preferred term label:**
Mesangial proliferative lupus nephritis

**Synonyms**
Class II lupus nephritis

**Definition (free text, please give PubMed ID)**
A lupus nephritis subclass with purely mesangial hypercellularity of any degree or mesangial matrix expansion by lightmicroscopy, with mesangial immune deposits. A few isolated subepithelial or subendothelial deposits may be visible by immunoflourescence or electron microscopy, but not by light microscopy.

**Parent term (use hpo.jax.org/app)**
Lupus nephritis (newly proposed term)

**Related terms** 
Mesangial hypercellularity HP:0012574
Mesangial Immune complex deposition HP:0033295
Systemic lupus erythematosus HP:0002725

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM #152700, ORPHA:536

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
